--- geocode-glib-0.99.2/geocode-glib/Makefile.am.orig	2013-08-15 17:24:18.000000000 +0200
+++ geocode-glib-0.99.2/geocode-glib/Makefile.am	2013-08-22 15:47:18.094343732 +0200
@@ -19,7 +19,7 @@
 	$(libgeocode_glib_la_PUBLICSOURCES)		\
 	geocode-glib-private.h
 
-libgeocode_glib_la_LIBADD = $(GEOCODE_LIBS) $(LIBS)
+libgeocode_glib_la_LIBADD = $(GEOCODE_LIBS) $(LIBS) -lm
 
 libgeocode_glib_la_LDFLAGS =				\
 	-version-info $(GCLIB_LT_VERSION)		\
